### ISSUE TITLE
[ESS][SUPPLY-CHAIN_ACT][TOP-DOWN] Script for  Demo Purpose  Showcasing the ESS Use Case without Business App

### ISSUE THREAD

**Issue by mkanal** [eclipse-tractusx/item-relationship-service#243]
**As** team
**I want** a script solution acting as a mock for a 3rd party business app for demo purposes 
**so that** can showcase the current development state  

## Hints / Details
* The Script or Insomnia routine should query the dDTR for all AAS registered for the requestor BPNL for the BOMLifecylce asPlanned 
* https://confluence.catena-x.net/display/IRS/%5BTRI-1519%5D%5BTRI-1523%5D+IRS%3A+ESS+Top-Down+Use+Case

### Outcome
- ...
### Acceptance Criteria
- [x] Script or Insomnia Collection is used to showcase the recursive traversal over a single part via the input of a key (single tuple of globalAssetId/bpnl pairs).
- [x] AAS count are limited to 5 AAS/jobs per request.  (Number of jobs in parallel is configurable )
- [x] The Script or Insomnia routine should query the dDTR for all AAS registered for the requestor BPNL for the BOMLifecylce asPlanned.
- [x] Script  or Insomnia Collection is used to showcase the recursive traversal over a batch of parts via the input of a multiple tuples of globalAssetId/bpnl pairs.
- [x] Scripts show response of requests (human-readable) to showcase the scenario step for step the demo participants 
- [x]  Script is prepared to showcase scenarios:
 - [x] [1] IncidentialSupplyChain(HappyPath): Complete supply chain is traversable. Incident BPNS is detected in supply chain. 
 - [x] [2] UnaffectedSupplyChain(HappyPath): Complete supply chain is traversable. Incident BPNS is NOT detected in supply chain.
 - [x] [3] IncidentialSupplyChain(SupplyChainContainsNOTCXMembers): Partial supply chain is traversable. Incident BPNS is detected in supply chain. Child in supply chain is not accessible because of any reason. 
 - [x] [4] UnknownSupplyChain(SupplyChainContainsNOTCXMembers):  Partial supply chain is traversable. Incident BPNS is NOT detected in supply chain.

## Out of Scope

- ...

**Comment by ds-jhartmann** [comment]
## Planning 2
- Create a python script
- figure out a way to have the four difference scenarios available to be tested
 - Case 1 and 2 can be done by choosing a different incident BPNS
 - Case 3 and 4 are aiming to test the case where a provider is not reachable
